packages/js/product-editor/src/components/editor/init-blocks.ts

```diff
@@ -38,20 +23,5 @@ export const initBlocks = () => {
 	// @ts-ignore An argument is allowed to specify which blocks to register.
 	registerCoreBlocks( blocks );
 
-	initImages();
-	initName();
-	initRadio();
-	initSummary();
-	initSection();
-	initTab();
-	initPricing();
-	initCollapsible();
-	initScheduleSale();
-	initTrackInventory();
-	initSku();
-	initConditional();
-	initLowStockQty();
-	initCheckbox();
-	initShippingDimensions();
-	initShippingFee();
+	Object.values( productBlocks ).forEach( ( init ) => init() );
```

Comment:
Nice reorganization here! 💯

Reply:
Yep is very tedious to configure a block, we need to touch many different places, definitely something to improve.